couchdb/tests/client.py

```diff
@@ -374,6 +374,76 @@ def test_query_multi_get(self):
         for idx, i in enumerate(range(1, 6, 2)):
             self.assertEqual(i, res[idx].key)
 
+    def test_find(self):
+        if self.server.major() < 2:
```

Comment:
I'd like to signal a skipped test for this case. I think you can do this by calling `self.skipTest()`, see [here](https://docs.python.org/2/library/unittest.html).